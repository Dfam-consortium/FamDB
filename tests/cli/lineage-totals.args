lineage
--format
totals
-ad
3
