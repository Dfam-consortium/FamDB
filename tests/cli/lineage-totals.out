1 entries in ancestors; 2 lineage-specific entries; found in partitions: 0;
